src/Networking.ino

```diff
@@ -135,7 +135,7 @@ void checkUDP()
       int len = portUDP.read(&packetBuffer[0], packetSize);
 
       if (len >= 2) {
-        if (packetBuffer[0] != 255)
+        if (reinterpret_cast<unsigned char&>(packetBuffer[0]) != 255)
```

Comment:
This one worries me a bit.
Not sure if we have obfuscated similar code where a char is cast into an `uint8_t` so PVS Studio doesn't see it.
On the other hand, maybe we can just trust PVS Studio to have done a thorough check.